ingest/src/test/java/com/microsoft/azure/kusto/ingest/AzureStorageClientTest.java

```diff
@@ -35,7 +40,7 @@ static void setUp() throws StorageException, URISyntaxException {
         testFilePath = Paths.get("src", "test", "resources", "testdata.json").toString();
         testFile = new File(testFilePath);
         testFilePathCompressed = Paths.get("src", "test", "resources", "testdata.json.gz").toString();
-        blob = new CloudBlockBlob(new URI("https://ms.com/storageUri"));
+        blob = new CloudBlockBlob(new URI("https://ms.com/storageUrl"));
```

Comment:
I wanted to see if it's an actual url, but it just redirects to the "Morgan-Stanley" bank lol. We might want to change it